lib/mayaUsd/ufe/UsdPointInstanceModifierBase.h

```diff
@@ -275,8 +342,27 @@ class MAYAUSD_CORE_PUBLIC UsdPointInstanceModifierBase
         return usdAttr;
     }
 
+    void _closeBatch()
+    {
+        // To close the batch we simply remove it from the map of batches under
+        // construction, and thus it can no longer be joined.  If a batch was
+        // created, erase() will harmlessly fail for all modifiers except the
+        // first one.  If no batch was created, make a trivial, unshared batch
+        // just for this modifier.
+        batches().erase(pointInstancerPath());
+        if (!_batch) {
+            _batch = std::make_shared<UsdPointInstanceBatch<UsdValueType>>();
+            _batch->nbInstances = 1;
+        }
+    }
+
+    // An alternative to storing these three data members would be to simply
```

Comment:
I really had to hold myself back with both hands to not change these three data members to a single UsdSceneItem, because that's exactly what a UsdSceneItem is: a prim, a path, and an instance index.  However, this kind of software engineering minor improvement has nothing to do with performance.